Taken from upstream binutils:
	* rl78-dis.c (print_insn_rl78): Use alternative form as a GCC false
	warning workaround.
  < http://sourceware.org/cgi-bin/cvsweb.cgi/src/opcodes/rl78-dis.c.diff?cvsroot=src&r1=1.3&r2=1.4 >
===================================================================
RCS file: /cvs/src/src/opcodes/rl78-dis.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- src/opcodes/rl78-dis.c	2012/08/15 22:37:56	1.3
+++ src/opcodes/rl78-dis.c	2013/04/10 19:16:45	1.4
@@ -217,7 +217,7 @@
 
 	    case '0':
 	    case '1':
-	      oper = opcode.op + *s - '0';
+	      oper = *s == '0' ? &opcode.op[0] : &opcode.op[1];
 	    if (do_es)
 	      {
 		if (oper->use_es && indirect_type (oper->type))
